src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/PropertyPages/BuildPropertyPage.VisualBasic.xaml

```diff
@@ -105,8 +109,12 @@
       <DataSource Persistence="ProjectFileWithInterception"
                   HasConfigurationCondition="False" />
     </EnumProperty.DataSource>
-    <EnumValue Name="On" DisplayName="On" />
-    <EnumValue Name="Off" DisplayName="Off" />
+    <EnumValue Name="On" DisplayName="On">
+      <!-- On: This is a VB keyword and should not be translated -->
```

Comment:
Are you expecting this comment to make its way to the .xlf/.lcl files?

Reply:
I am not expecting it to currently be included in the xlf files, but I do know the syntax to do that is similar. I wasn't able to get them to show up, but since I still kept the keyword translations marked as "translated," our loc team may not need this context.